frontend/src/core/placeable/parsers/pythonFormattingVariable.test.js

```diff
@@ -15,15 +15,28 @@ describe('pythonFormattingVariable', () => {
         ['%(cows.number)d', 'There were %(cows.number)d cows'],
         ['%(number of cows)d', 'There were %(number of cows)d cows'],
         ['%(number)03d', 'There were %(number)03d cows'],
-        ['%(number) 3d', 'There were %(number) 3d cows'],
         ['%(number)*d', 'There were %(number)*d cows'],
         ['%(number)3.1d', 'There were %(number)3.1d cows'],
         ['%(number)Ld', 'There were %(number)Ld cows'],
+        ['%s', 'path/to/file_%s.png'],
+        ['%s', 'path/to/%sfile.png'],
```

Comment:
I'm curious - why did you decide to add this test case? Is it related to the patch?

Reply:
Because I was learning more about regexp and thought that what I was doing could break this specific case, which I didn't want to happen. Figured I might as well leave that around.

Reply:
OK, makes sense!